view/frontend/templates/widget/posts.phtml

```diff
@@ -0,0 +1,179 @@
+<?php
+/**
+ * Mageplaza
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Mageplaza.com license that is
+ * available through the world-wide-web at this URL:
+ * https://www.mageplaza.com/LICENSE.txt
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade this extension to newer
+ * version in the future.
+ *
+ * @category    Mageplaza
+ * @package     Mageplaza_Blog
+ * @copyright   Copyright (c) 2016 Mageplaza (http://www.mageplaza.com/)
+ * @license     https://www.mageplaza.com/LICENSE.txt
+ */
+?>
+<?php
+$posts = $block->getCollection();
+?>
+
+
+    <div class="post-list-container mpcss">
+        <div class="row">
+            <?php if ($block->getBlogConfig('sidebar/sidebar_left_right')==1){
+				?>
```

Comment:
- Php closing tag should go onto previous line.
- I believe that getBlogConfig will return a string, if so please change line to `$block->getBlogConfig('sidebar/sidebar_left_right') === '1'`